zulipterminal/api_types.py

```diff
@@ -244,9 +250,11 @@ class Subscription(Stream):
 
     is_muted: bool
 
-    role: int  # NOTE: new in Zulip 4.0 / ZFL 31
     color: str
 
+    # Deprecated fields
+    # role: int  # Removed in Zulip 6.0 (feature level 133)
```

Comment:
I think I noted somewhere that it's worth including here that it wasn't actively used by us (or the web app even?), and introduced at some other ZFL?